package.json

```diff
@@ -121,7 +121,7 @@
     "css-loader": "^0.19.0",
     "devtron": "^1.1.0",
     "dom-storage": "^2.0.2",
-    "electron": "2.0.7",
+    "electron": "2.0.9",
```

Comment:
@crakoucas can you update it to the lastest one. Thx

Reply:
No problem